app/model_config.py

```diff
@@ -114,24 +117,24 @@ def vocab(self):
         return PathConfig(self.get_property("vocab"), self.model_name)
 
 
-OUT = {
-    "base_dir": os.path.join(os.getcwd(), "classifier_models"),
-    "model_name": "UPR_2percent_ps0",
-    "instance_name": "1573031002",
-    "saved_model": {
-        "name": "saved_model.pb",
-        "path": "multilabel/{model}/saved_model/{instance}"
-    },
-    "variables": {
-        "path": "multilabel/{model}/saved_model/{instance}/variables"
-    },
-    "vocab": {
-        "name": "label.vocab",
-        # NOTE: We store the vocab file inside an instance folder
-        # in case it changes between training runs.
-        "path": "multilabel/{model}/saved_model/{instance}"
-    },
-}
+# OUT = {
```

Comment:
remove now that ./static/...json exists?

Reply:
done